src/microbe_stage/editor/CellStatsIndicator.cs

```diff
@@ -91,14 +98,17 @@ public float Value
     public override void _Ready()
     {
         descriptionLabel = GetNode<Label>("Description");
-        valueLabel = GetNode<Label>("Value");
+        valueLabel = GetNode<Label>(ValuePath);
```

Comment:
I'm slightly concerned about the impact of this change to a nodepath being set through the editor here (in terms of performance as a lot of cell stats indicators are used). With the switch to Godot 4 this situation should get resolved anyway then at the latest.